pkg/defaults/mesh/signing_key.go

```diff
@@ -0,0 +1,20 @@
+package mesh
+
+import (
+	"context"
+
+	"github.com/kumahq/kuma/pkg/core/resources/manager"
+	core_store "github.com/kumahq/kuma/pkg/core/resources/store"
+	"github.com/kumahq/kuma/pkg/tokens/builtin/issuer"
+)
+
+func createSigningKey(resManager manager.ResourceManager, meshName string) error {
+	key, err := issuer.CreateSigningKey()
+	if err != nil {
+		return err
+	}
+	if err := resManager.Create(context.Background(), &key, core_store.CreateBy(issuer.SigningKeyResourceKey(meshName))); err != nil {
```

Comment:
Owners automatically set on ScopeMesh resources in regular ResourceManager: `pkg/core/resources/manager/manager.go`. But secret's Create apparently goes through `customizableResourceManager` and I'm not sure that the owner will be set automatically

Reply:
It won't https://github.com/kumahq/kuma/issues/1137